React InstantSearch Hooks Native/getting-started/src/Filters.tsx

```diff
@@ -0,0 +1,164 @@
+          <View style={styles.itemCount}>
+            <Text style={styles.itemCountText}>{totalRefinements}</Text>
+          </View>
+        )}
+      </TouchableOpacity>
+
+      <Modal animationType="slide" visible={isModalOpen}>
+        <SafeAreaView>
+          <View style={styles.container}>
+            <View style={styles.title}>
+              <Text style={styles.titleText}>Brand</Text>
+            </View>
+            <View style={styles.list}>
+              {items.map((item) => {
+                return (
+                  <TouchableOpacity
+                    key={item.value}
+                    onPress={() => {
+                      refine(item.value);
+                      onChange();
+                    }}
+                    style={styles.item}
+                  >
+                    <Text
+                      style={{
+                        ...styles.labelText,
+                        fontWeight: item.isRefined ? '800' : '400',
+                      }}
+                    >
+                      {item.label}
+                    </Text>
+                    <View style={styles.itemCount}>
+                      <Text style={styles.itemCountText}>{item.count}</Text>
+                    </View>
+                  </TouchableOpacity>
+                );
+              })}
+            </View>
+          </View>
+          <View style={styles.filterListButtonContainer}>
+            <View style={styles.filterListButton}>
+              <Button
+                onPress={() => {
+                  clear();
+                  onChange();
+                  onToggleModal();
+                }}
+                title="Clear all"
+                color="#252b33"
+                disabled={!canClear}
+              />
+            </View>
+            <View style={styles.filterListButton}>
+              <Button
+                onPress={onToggleModal}
+                title="See results"
+                color="#252b33"
+              />
+            </View>
+          </View>
+        </SafeAreaView>
+      </Modal>
+    </>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    padding: 18,
+    backgroundColor: '#ffffff',
+  },
+  title: {
+    alignItems: 'center',
+  },
+  titleText: {
+    fontSize: 32,
+  },
+  list: {
+    marginTop: 32,
+  },
+  item: {
+    paddingVertical: 12,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    borderBottomWidth: 1,
+    borderColor: '#ddd',
+    alignItems: 'center',
+  },
+  itemCount: {
+    backgroundColor: '#252b33',
+    borderRadius: 24,
+    paddingVertical: 4,
+    paddingHorizontal: 8,
+    marginLeft: 4,
+  },
+  itemCountText: {
+    color: '#ffffff',
+    fontWeight: '800',
+  },
+  labelText: {
+    fontSize: 16,
+  },
+  filterListButtonContainer: {
+    flexDirection: 'row',
+  },
+  filterListButton: {
+    flex: 1,
+    alignItems: 'center',
+    marginTop: 18,
+  },
+  filterListButtonText: {
+    fontSize: 18,
+  },
```

Comment:
It looks like it's not used.

```suggestion
```

Reply:
Indeed, addressed in [479dd92](https://github.com/algolia/doc-code-samples/pull/358/commits/479dd923dec3d3dd2347e5c99e54f35b61b24122).